activerecord/test/schema/postgresql_specific_schema.rb

```diff
@@ -40,6 +40,8 @@
     char2 character varying(50) default 'a varchar field',
     char3 text default 'a text field',
     bigint_default bigint default 0::bigint,
+    uuid_default uuid default 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11',
```

Comment:
Instead of modifying the global schema, could you create the necessary table and columns directly in the test-case itself and then remove it afterwards?